scripts/alert_emails_report/generate-report.ts

```diff
@@ -81,6 +84,39 @@ async function updateSubscriptionsByLocation() {
   await gsheets.clearAndAppend(spreadsheetId, RANGE_STATES, stateStats);
 }
 
+async function updateSubscriptionsByDate(subscriptions: Subscription[]) {
+  const gsheets = new GoogleSheets(keyFile);
+  const spreadsheetId = getSpreadsheetId();
+
+  const sortedSubscriptions = _.chain(subscriptions)
+    .filter(s => s.subscribedAt !== undefined)
+    .sortBy(s => s.subscribedAt)
+    .value();
+
+  const dateTotals = [];
```

Comment:
Maybe this can be simplified a bit?

```
const dateTotals = _.chain(subscriptions)
  .filter(s => s.subscribedAt !== undefined)
  .groupBy(s => moment(s.subscribedAt).format('YYY-MM-DD'))
  .map((subscriptionList, dateKey) => [dateKey, subscriptionList.length])
  .value()
```

Reply:
While this is much cleaner (thank you!), it ends up computing daily deltas instead of daily cumulatives which means there's an extra step involved to turn it into the data we can graph. So I've opted to leave the existing code for now, since I haven't gotten a chance to revisit this. Thanks / sorry!